OracleSiebel/OEL/build/multi-stage/cache/dockerfile

```diff
@@ -0,0 +1,81 @@
+# Globals
+
+ARG VERSION=19.03
+ARG BUILD_DATE=2019-04-30
+
+
+FROM oraclelinux:latest as temp
+ARG INSTALLSITE=installers
+ARG VERSION
+
+COPY oraInst.loc getBase.sh getUpdate.sh cache/cache_base.rsp cache/cache_install_base.sh cache/cache_update.rsp.templ cache/cache_install_update.sh siebelkeystore.jks /config/
+
+# Install Siebel Base
+
+ENV LANG=en_US.UTF-8
+
+RUN mkdir -p -ma+rwx /siebel/cache /siebel/oraInventory /stage \
+&& groupadd -g 1000 siebel && useradd -m -g siebel -G siebel -u 1000 siebel \
+&& yum -y install wget tcsh ksh \
+&& cd /mnt \
+&& date \
```

Comment:
Why are you calling date multiple times? If it's to determine how long it takes to download the installer, just prepend that command with `time`, i.e. `time bash /config/getBase.sh ${INSTALLSITE}`

Reply:
Thanks for the better method. Changed accordingly